main.py

```diff
@@ -211,21 +229,32 @@ def _get_owners_id(owners):
 
 def _get_owners(client, project):
     users = []
-    project_id = project.get('projectId')
-    iamPolicy = client.projects().getIamPolicy(resource=project_id, body={}).execute()
+    project_name = project.get('name')
+    iamPolicy = client.projects().getIamPolicy(resource=project_name, body={}).execute()
     bindings = iamPolicy.get('bindings', [])
     owners = list(filter(filter_owners, bindings))
     if not owners:
-        logger.debug('No owners found for Project %s.', project_id)
+        logger.debug('No owners found for Project %s.', project_name)
     else:
         members = owners[0].get('members')
         users = list(filter(filter_users, members))
         if not users:
-            logger.debug('No owner is a user for Project %s.', project_id)
+            logger.debug('No owner is a user for Project %s.', project_name)
     users = set([user.strip('user:') for user in users])
     return list(users)
 
 
+def _get_organization(client, project):
+    client = client = discovery.build("cloudresourcemanager", "v1")
```

Comment:
Initialize the client only once: move it out of the loop, create a method like the `_get_resource_manager_client`.
Give a distinct name for the client: client_v1 or something like that.

Reply:
- client_v1 is initialized only once, in a method like the original `_get_resource_manager_client` 14a6234